### DISCUSSION TITLE
projet mort ?

### DISCUSSION THREAD

**Discussion by BujardJoaillier** [sidhq/Multi-GPT#11] [Q&A] [answered]
plus de mise à jours ? que ce passe t'il ?

**Comment by ruflair** [comment, marked as answer]
Hi  @BujardJoaillier!
We have been busy working on something else and we have had very little time to work on MultiGPT these past few weeks unfortunately. You can learn more about what we are building [here](https://sid.ai/).
We have some exciting news coming up soon!
